src/main/k/in-progress/tests/priority/tests/1.output.golden

```diff
@@ -0,0 +1,3 @@
+<k>
+  #Exists _0 . _0 : #Exists _0 . _0 : #Exists _0 . _0 : .Seq
```

Comment:
Is this the right output?

Reply:
no. I just forgot to update it. thanks for mentioning it.